proto/iotics/api/meta.proto

```diff
@@ -29,6 +29,12 @@ service MetaAPI {
   //   choose to end the stream early once they have received enough results. (E.g. in the case of Scope.LOCAL this
   //   would be after the one and only sequence of chunks has been received.)
   rpc SparqlQuery(SparqlQueryRequest) returns (stream SparqlQueryResponse) {}
```

Comment:
Here we might want to offer 2 ways of using the new service:
- a gRPC stream/unary service
- a gRPC unary/unary service (for the smaller payload) - maybe not required for the first implementation?

Reply:
stream/unary covers the small payload case - do we really want/need to introduce another rpc? 
if the goal is to simplify that smaller payload case, i don't think this will be much benefit right now

Reply:
I agree that if we start with the client-stream one, it should be pretty easy to add support for the unary variant (with v. minor additions in gRPC service impl.)

Reply:
I agree with your comments, stream/unary is a good starting point and it will cover the small payload case.

We need to validate with @mihai-dinculescu that grpc-web can deal with stream/unary services and also consider that for the end-user it might be way easier to run a unary/unary call than a stream/unary call with chunking.